css/style.css

```diff
@@ -0,0 +1,51 @@
+.graybackground{
+  background-color: rgb(234, 229, 229);
+}
+main{
+  display: block;
+  float:right;
```

Comment:
Please insert a single space after " : ".

Reply:
- [ ] Kindly explain the why of your comments to make sure the person you are reviewing understand why they're going to make this changes :+1: